examples_tests/35.TAA/main.cpp

```diff
@@ -300,7 +275,7 @@ int main()
 		driver->beginScene(false, false, video::SColor(255,255,255,255) );
 
         //force view and projection matrices update
-        camera->render();
+        camera->render();//this sets view matrix to all nans in 1st frame...
```

Comment:
you also need `OnAnimate()` that actually calculates the matrices from mouse and keyboard

`render()` only sets them in the driver